### ISSUE TITLE
Plugin requires SetOption65 to be 0 to work, but my plug keeps resetting if I do that.

### ISSUE THREAD

**Issue by Deses** [jneilliii/OctoPrint-Tasmota#196]
I flashed a smart plug model BSD34 and if I don't do the [SetOption65 1](https://tasmota.github.io/docs/Commands/#setoption65) to disable Fast Power Cycle Device Recovery, my plug keeps reseting and I'm unable to do anything with it.

Is it absolutely necessary to have to disable SetOption65 to make the plugin work?

**Comment by jneilliii** [comment]
I don't even know what that option is, so shouldn't effect the plugin.

**Comment by jneilliii** [comment]
The only setting the plugin prefers is SetOption26, but can run without that being set if you use a blank index.

**Comment by Deses** [comment]
Sorry I left this hanging for two weeks. I got frustrated with my smart plug and left it on my desk collecting dust for that time. 

Anyway I gave it another go, resoldered everything just in case anything was wrong and tried the setup process again, and saw that the option requires is indeed SetOption26, not 65. I guess that in my frustration misread that and blamed my problems on SetOption26.

Thank you @jneilliii for your patience and sorry for the useless issue! ❤️
